modules/secrets-store-sync-secrets.adoc

```diff
@@ -0,0 +1,80 @@
+// Module included in the following assemblies:
+//
+// * nodes/pods/nodes-pods-secrets-store.adoc
+
+:_content-type: PROCEDURE
+[id="secrets-store-sync-secrets_{context}"]
+= Enabling synchronization of mounted content as Kubernetes secrets
+
+You can enable synchronization to create Kubernetes secrets from the content on the mounted volume. An example where you might want to enable synchronization is to use an environment variable in your deployment to reference the Kubernetes secret.
+
+[WARNING]
+====
+Do not enable synchronization if you do not want to store your secrets on your {product-title} cluster and in etcd. Enable this functionality only if you require it; for example, to use environment variables to refer to the secret.
+====
+
+// TODO: If they enable sync, should we recommend they enable auto rotation too, or can they be done independent of each other?
```

Comment:
Auto rotation is enabled by default and can not be disabled at least for TP.

Reply:
Thanks, that was an old question before I found out that it wouldn't be optional